index.html

```diff
@@ -6,128 +6,133 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./styles.css" />
     <title>Lenz Lösches Homepage</title>
-    <script src="script.js"></script>
+    <script src="script.js" defer></script>
   </head>
   <body id="body">
     <header></header>
     <nav>
-      <div class="navdiv" id="navdiv">
-        <h3 class="navelement"><a href="#body">Home</a> -</h3>
+      <ul class="nav" id="navdiv">
+        <li class="nav__element"><a href="#body">Home</a> -</li>
 
-        <h3 class="navelement"><a href="#lebenslauf">About</a> -</h3>
+        <li class="nav__element"><a href="#lebenslauf">About</a> -</li>
 
-        <h3 class="navelement"><a href="#skills">Skills</a> -</h3>
+        <li class="nav__element"><a href="#skills">Skills</a> -</li>
 
-        <h3 class="navelement"><a href="#links">Links</a> -</h3>
+        <li class="nav__element"><a href="#links">Links</a> -</li>
 
-        <h3 class="navelement"><a href="#kontakt">Contact</a></h3>
+        <li class="nav__element"><a href="#kontakt">Contact</a></li>
 
-        <div class="arrow" id="arrow">
-          <img class="arrowimg" src="bilder/arrow.gif" alt="" />
+        <div class="nav__arrow" id="arrow">
+          <img class="nav__arrow-img" src="bilder/arrow.gif" alt="Arrow" />
         </div>
-      </div>
+      </ul>
     </nav>
     <main>
-      <div class="containerdiv background-welcome positionrelative">
+      <section class="containerdiv background-welcome positionrelative">
         <br /><br />
         <br /><br />
         <figure align="center" class="figure">
-          <img src="/bilder/lenz.jpg" class="image" />
+          <img
+            src="/bilder/lenz.jpg"
+            class="lenz-image"
+            alt="Lenz Lösche sitting at desk"
+          />
         </figure>
-      </div>
+      </section>
       <div class="line"></div>
 
-      <div class="containerdiv background-vita" align="center" id="lebenslauf">
-        <div class="vitatable">
-          <div class="about">
+      <section
+        class="containerdiv background-vita"
+        align="center"

```

Comment:
The `align` attribute is deprecated (https://www.w3.org/TR/html401/present/graphics.html).
Please use the CSS `text-align` property to achieve the effect.